kdumpctl

```diff
@@ -962,8 +962,22 @@ check_dump_feasibility()
 	check_kdump_feasibility
 }
 
+fadump_bootargs_append()
+{
+	if [[ -f "$FADUMP_APPEND_ARGS_SYS_NODE" ]]; then
+		output=$( { echo "${FADUMP_COMMANDLINE_APPEND}" > "$FADUMP_APPEND_ARGS_SYS_NODE" ; } 2>&1)
+		if [ $? -eq 0 ]; then
+			output=$(cat "$FADUMP_APPEND_ARGS_SYS_NODE")
+			dinfo "fadump: additional parameters for capture kernel: '$output'"
+		else
+			dwarn "WARNING: failed to setup additional parameters for fadump capture kernel: '$output'"
+		fi
+	fi
+}
```

Comment:
noticed an other small nit. Could you please add a warning in case $FADUMP_APPEND_ARGS_SYS_NODE does not exist telling the user, that the current kernel does not support setting additional parameters.

Reply:
Thanks for the review. Updated the patch accordingly.